daml/splice-amulet/daml/Splice/AmuletRules.daml

```diff
@@ -437,11 +446,14 @@ template AmuletRules
         summary : OpenMiningRoundSummary
       controller dso
       do
+        now <- getTime
         miningRound <- fetchAndArchive (ForDso with dso) miningRoundCid
         validateOpenMiningRoundSummary summary
 
-        let params = computeIssuingRoundParameters
-              miningRound.tickDuration miningRound.amuletPrice miningRound.issuanceConfig summary
+        let
+          optDevelopmentFundPercentage = (.percentage) <$> (getValueAsOf now configSchedule).developmentFundConfig
```

Comment:
We generally try to pin all parameters for a round as part of round creation to avoid surprises when issuance configs change.

With my proposal above of putting `developmentFundPercentage` into `IssuanceConfig` you get this for free.